Frontend/ui-customer/src/Flow.purs

```diff
@@ -2492,6 +2496,7 @@ editDestinationFlow = do
     callCheckRideStatus homeScreenState = do
       modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen{data{destination = homeScreenState.data.driverInfoCardState.destination, destinationAddress = homeScreenState.data.driverInfoCardState.destinationAddress}, props{destinationLat = homeScreenState.data.driverInfoCardState.destinationLat, destinationLong = homeScreenState.data.driverInfoCardState.destinationLng}})
       setValueToLocalStore TRACKING_DRIVER "False"
+      void $ lift $ lift $ toggleLoader true
```

Comment:
if we need to enable loader before calling the API in checkRideStatus we can add this line inside checkRideStatus itself, at just a single place.

Reply:
this function is used at multiple places other than edit dest use cases. I didn't want to un-necessary add it to the other places.

Reply:
I think it's better to add it in the function itself. Before making the API call we can enable the loader then disable after getting the response.